# HG changeset patch
# User Rik <rik@octave.org>
# Date 1470617623 25200
#      Sun Aug 07 17:53:43 2016 -0700
# Node ID 6bb0f32d22a55ff23a3e9e866becc3b68d700a90
# Parent  3ac380d46d5481f9e423d070a6da58d7a7214662
axis.m: Clean up formatting in %!error tests.

* axis.m: Clean up formatting in %!error tests.

diff --git a/scripts/plot/appearance/axis.m b/scripts/plot/appearance/axis.m
--- a/scripts/plot/appearance/axis.m
+++ b/scripts/plot/appearance/axis.m
@@ -640,23 +640,23 @@ endfunction
 %!   axis tight;
 %!   assert (axis (), [1 49 1 29 min(Z(:)) max(Z(:))]);
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
 
 ## Even on errors, axis can display a figure.
 
-%!error<LIMITS vector must have .* elements>
-%! hf= figure ("visible", "off");
+%!error <LIMITS vector must have .* elements>
+%! hf = figure ("visible", "off");
 %! unwind_protect
 %!   axis (1:5)
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
 
-%!error<expecting no args, or a numeric vector with .* elements>
-%! hf= figure ("visible", "off");
+%!error <expecting no args, or a numeric vector with .* elements>
+%! hf = figure ("visible", "off");
 %! unwind_protect
 %!   axis ({1,2})
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
